packages/babel-traverse/src/path/modification.js

```diff
@@ -112,7 +116,10 @@ export function insertAfter(nodes) {
     this.parentPath.isExpressionStatement() ||
     this.parentPath.isLabeledStatement()
   ) {
-    return this.parentPath.insertAfter(nodes);
+    // `replaceWithMultiple` requeues if there's a replacement for this.node,
+    // but not for an ancestor's. Set `shouldRequeue` to true so that any replacement
+    // for an ancestor node can be enqueued. Fix #5628 and #5023.
+    return this.parentPath._insertAfter(nodes, true);
```

Comment:
But I can invoke `#insertBefore` [myself and get the same error](http://astexplorer.net/#/gist/47c24345d741602dc04928b353fdd254/515de002dc703383abd0ca91ca18faa812b076e2). [Here](https://github.com/babel/babel/pull/5743/files#diff-04e63519c0e77a22a1a20ef218acc048R32), right?